cmd/network/show.go

```diff
@@ -56,6 +58,11 @@ var showCmd = &cobra.Command{
 		consensusConn := conn.Consensus()
 		registryConn := consensusConn.Registry()
 
+		ctrlConn := conn.Control()
+		nodeStatus, err := ctrlConn.GetStatus(ctx)
```

Comment:
Why are you getting committees from the node status? Then you are limited only to the runtimes that a given node has configured while other runtimes may exist. In order to support all runtimes, you could get a list of all runtimes from the registry via `registryConn.GetRuntimes(...)`. And you could skip any suspended runtimes that don't have committees.

Reply:
Ok, now I am getting a list of all runtimes from the registry, and runtime state from `roothashConn`. Hope it is correct.